client/lib/analytics/ad-tracking/woo/record-gtm-datalayer-event.ts

```diff
@@ -0,0 +1,38 @@
+import { mayWeTrackByTracker } from '../../tracker-buckets';
+import { debug, TRACKING_IDS } from '../constants';
+import { initGTMContainer, loadGTMContainer } from '../gtm-container';
+
+/**
+ * Sends a step event to Google Tag Manager for the woo express stepper flow.
+ *
+ * @param {string} stepName
+ * @returns {void}
+ */
+export default async function recordGTMDatalayerEvent( stepName = '' ) {
```

Comment:
```
export default function recordGTMDatalayerEvent( stepName: string ): void {
```

but if you want the function to be `async` you can't return `void`, then it has to return a promise, and also you need to `await` for it to resolve/reject where you use the function. I don't think we need it to be async in this case.

Reply:
@gmovr  - Apologies, that was indeed a typo :-) I copied the other method and was building this off of that other piece of code. Just before creating the PR, I might have done an undo and the async got pushed up. Thanks for spotting it.